examples/src/main/python/ml/simple_params_example.py

```diff
@@ -36,18 +35,20 @@
     if len(sys.argv) > 1:
```

Comment:
We're moving most examples towards being more simple (with a few exceptions, such as keeping the longer ML examples that show a bit how to build an app and use args parsing). As such I agree we should remove this.